dpnp/dpnp_iface_manipulation.py

```diff
@@ -73,7 +73,7 @@
 ]
 
 
-def asfarray(x1, dtype=numpy.float64):
+def asfarray(x1, dtype=None):
     """
     Return an array converted to a float type.
 
```

Comment:
Add a limitation note for `dtype` parameter and its default value, since it is different from numpy description of `asfarray`